dali/operators/sequence/optical_flow/optical_flow.h

```diff
@@ -41,6 +41,7 @@ struct backend_to_compute<GPUBackend> {
 
 const std::string kPresetArgName = "preset";                               // NOLINT
 const std::string kOutputFormatArgName = "output_format";                  // NOLINT
+const std::string kHintFormatArgName = "hint_format";                      // NOLINT
```

Comment:
I don't think these fields were used a lot (if at all) given that previously there was just one legal value (4) - and even in the documentation this was referred to as "grid_size", not "output_format". I think we should use "output_grid" and "hint_grid", respectively or something along these lines. In any case, it's not "format", since the format is the same - a pair of numbers in x, y order.

Reply:
Done